app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from app.errors import NotWearingMaskError, VaccineError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> str:
+    without_mask_count = 0
+    friends_count = 0
+    count = 0
+    for friend in friends:
+        count += 1
+        try:
+            if not friend["wearing_a_mask"]:
+                without_mask_count += 1
+            cafe.visit_cafe(friend)
+            friends_count += 1
+            if without_mask_count != 0 and friend == friends[-1]:
+                raise NotWearingMaskError
```

Comment:
why do u raise exception here? `cafe.visit_cafe(friend)` raises already expetion if something wrong

Reply:
without this if part exception not raise